examples/extensions-zalando/src/test/java/org/zalando/intellij/swagger/examples/extensions/zalando/validator/zally/MockZallySettings.java

```diff
@@ -13,4 +13,14 @@ public String getZallyUrl() {
   public void setZallyUrl(String zallyUrl) {
     // Not needed for a mock, no test cases for it.
   }
+
```

Comment:
Why do you prefer not to use a mocking library (e.g. Mockito)?

Reply:
I suppose I _could_ use Mockito, but this service class is actually injected to the in-memory IDE when executing tests via https://github.com/zalando/intellij-swagger/blob/master/examples/extensions-zalando/src/main/resources/META-INF/plugin.xml#L38. I don't think Mockito would bring extra value in this case.